components/pdf_preview.jsx

```diff
@@ -148,7 +150,7 @@ export default class PDFPreview extends React.PureComponent {
         for (let i = 0; i < this.state.numPages; i++) {
             pdfCanvases.push(
                 <canvas
-                    ref={'pdfCanvas' + i}
+                    ref={this.pdfCanvas[i]}
```

Comment:
does this work correctly? I think you need to create the references dinamically for it to work, otherwise it might fail as I don't think it is an array

Reply:
fixed